UberBarber/UberBarber/database/DatabaseQueries.cs

```diff
@@ -69,14 +70,35 @@ public string User_validation(string username, string password, string confirm_p
             // Returns proper message when the validation is correct or not.
         {
             string message = "Something went wrong :(";
+            string usernamePattern = "^[a-zA-Z0-9.]{6,}$";
+            string passwordPattern = "^[a-zA-Z]{6,}$";
```

Comment:
is this a pattern for only letters in password? add numbers here

Reply:
Numbers added to the pattern.